app/assets/src/components/views/SampleViewV2/ReportFilters.jsx

```diff
@@ -0,0 +1,275 @@
+            <SearchBox
+              clearOnSelect
+              rounded
+              levelLabel
+              serverSearchAction="choose_taxon"
+              serverSearchActionArgs={{
+                // TODO (gdingle): change backend to support filter by sampleId
+                args: "species,genus",
+                sample_id: sampleId,
+              }}
+              onResultSelect={(_, { result }) => {
+                return this.handleFilterChange({
+                  key: "taxon",
+                  value: {
+                    taxId: result.taxid,
+                    taxLevel: result.level,
+                    name: result.title,
+                  },
+                });
+              }}
+              placeholder="Taxon name"
+            />
+          </div>
+          <div className={cs.filterListElement}>
+            <NameTypeFilter
+              value={selected.nameType}
+              onChange={value =>
+                this.handleFilterChange({
+                  key: "nameType",
+                  value,
+                })
+              }
+            />
+          </div>
+          {/* from server */}
+          <div className={cs.filterListElement}>
+            <BackgroundModelFilter
+              allBackgrounds={backgrounds}
+              value={selected.background}
+              onChange={value =>
+                this.handleFilterChange({
+                  key: "background",
+                  value,
+                })
+              }
+            />
+          </div>
+          {/* from server */}
+          <div className={cs.filterListElement}>
+            <CategoryFilter
+              allCategories={CATEGORIES}
+              categoryParentChild={transform((result, category) => {
+                if (category.children)
+                  result[category.name] = category.children;
+              }, {})(CATEGORIES)}
+              categoryChildParent={transform((result, category) => {
+                forEach(
+                  subcat => (result[subcat] = category.name),
+                  category.children || []
+                );
+              }, {})(CATEGORIES)}
+              selectedCategories={getOr(
+                [],
+                ["categories", "categories"],
+                selected
+              )}
+              selectedSubcategories={flatten(
+                values(getOr({}, ["categories", "subcategories"], selected))
+              )}
+              onChange={(categories, subcategories) =>
+                this.handleFilterChange({
+                  key: "categories",
+                  value: {
+                    categories,
+                    subcategories,
+                  },
+                })
+              }
+            />
+          </div>
+          <div className={cs.filterListElement}>
+            <ThresholdFilterDropdown
+              options={{
+                targets: THRESHOLDS,
+                operators: [">=", "<="],
```

Comment:
Might have missed this on the previous PR, but threshold filters seem to be acting pretty weirdly...

**Filtering for score >= 10 affects the upper bound for score instead of lower bound**
![image](https://user-images.githubusercontent.com/53838890/69392967-b29fe700-0c8c-11ea-8e72-2e7ae13b8847.png)
![image](https://user-images.githubusercontent.com/53838890/69392973-b5024100-0c8c-11ea-92c2-26850950abb4.png)

**Filtering for score >= 1000 doesn't actually seem to filter any taxons out**
![image](https://user-images.githubusercontent.com/53838890/69392830-30172780-0c8c-11ea-99ee-972ba459d6c9.png)
![image](https://user-images.githubusercontent.com/53838890/69392837-360d0880-0c8c-11ea-8949-a0e5549d51f6.png)

**Filtering for score <= 1000 behaves as if filtering for >= 1000 instead**
![image](https://user-images.githubusercontent.com/53838890/69392871-51781380-0c8c-11ea-8d39-720a24b97920.png)
![image](https://user-images.githubusercontent.com/53838890/69392873-53da6d80-0c8c-11ea-8e88-03eb3c149c5d.png)

Reply:
Thanks for catching. They were reversed. Fixed.